### ISSUE TITLE
Пропуск стратегии при первой неудачной попытке

### ISSUE THREAD

**Issue by RocketRide9** [bol-van/zapret#1000]
Здравствуйте. В 69.9 blockcheck.sh прогоняет ровно то количество попыток, которое было указано пользователем. Прошу добавить пропуск тестирования текущей стратегии, если какая-либо попытка была неуспешна. Это сильно сэкономит время тестирования стратегий.

Я же правильно понимаю, что нету смысла пробовать пользоваться стратегией, где хоть одна попытка была неуспешна?

**Comment by bol-van** [comment]
На этот вопрос отвечал много раз
Блокчек создавался не для копипасты, а для исследования DPI. Там имеет значение все.
И что он выдает, и какова частота неудач
Потому разве что имеет смысл сделать отдельный режим. Или сразу делать распараллеливание по форкам

**Comment by NewUse** [comment]
да, работа в параллель или асинхрон была бы крайне кстати... а то с учётом множества попыток, время прогона возрасло кратно по сравнению с начальными версиями

**Comment by bol-van** [comment]
PARALLEL=1 сделано, но как показывает практика многие сайты воспринимают это как флуд и начинают плохо себя вести
